internal/common/ingest/testfixtures/event.go

```diff
@@ -18,16 +16,21 @@ import (
 // Standard Set of events for common tests
 const (
 	JobIdString                  = "01f3j0g1md4qx7z5qb148qnh4r"
+	JobIdString2                 = "01f3j0g1md4qx7z5qb148qnh5r"
```

Comment:
Duplicating all these is probably not the best approach. Have a look at what we do, e.g., for the GangScheduler tests. A similar constructor approach would probably be nicer here.